apps/server/src/shared/domain/interface/rolename.enum.ts

```diff
@@ -9,6 +9,8 @@ export enum RoleName {
 	DEMOTEACHER = 'demoTeacher',
 	EXPERT = 'expert',
 	HELPDESK = 'helpdesk',
+	ROOM_VIEWER = 'room_viewer',
+	ROOM_EDITOR = 'room_editor',
```

Comment:
Is snake_case the right convention for enum values?

Reply:
Yes you are right. The keys can be reversed.

But the values are already written. Hmm. but we can migrate them.